spec/services/metrics_service_spec.rb

```diff
@@ -109,7 +112,8 @@
           expect(all_metric_dataset).to receive(:put).with(
             [
               {
-                average_completion_time: 1515.0,
+                mode_completion_time_controlled: 120.0,
+                mode_completion_time_certificated: 120.0,
```

Comment:
can I suggest making one of these two number different or is that too fiddly with the `created_at` dates? Just to distinguish clearly that the correct value is being returned